Alignment/CocoaModel/src/CocoaDaqReaderRoot.cc

```diff
@@ -128,7 +129,7 @@ OpticalAlignMeasurementInfo CocoaDaqReaderRoot::GetMeasFromPosition2D( AliDaqPos
   OpticalAlignMeasurementInfo meas;
   
   meas.type_ = "SENSOR2D";
-  meas.name_ = pos2D->GetID();
```

Comment:
Implicit conversion is possible. Thus the following would work:
```
meas.name_ = pos2D->GetID().Data();
```
Which is basically the same as `std::string(pos2D->GetID().Data())`, because `std::string` has constructor which takes `const char *` and compiler can map that.

The problem is that `TString` has conversion functions to `const char *` and `std::string_view`, but the same are also available in `std::string` constructors. This means means we have two ways to constructor a string (or do things like +=) and they have the same priority. For compiler that's ambiguity -- it doesn't know which to pick. For example, the one below would force the compiler to pick `const char *` conversion operator of `TString`:
```
meas.name_ = static_cast<const char *>(pos2D->GetID());
``` 
As you can see there multiple ways you can write this. I depends how much you want to be explicit about what needs to happen or not, but end result (executed code) should be the same. One could write full chain explicitly instead of implicitly done by compiler:
```
meas.name_ = std::string(static_cast<const char *>(pos2D->GetID()));
```